detection_rules/rule.py

```diff
@@ -966,18 +983,30 @@ def get_version_space(self) -> Optional[int]:
 
     @property
     def latest_version(self) -> Optional[int]:
-        """Retrieve the latest known version of the rule."""
-        min_stack = self.get_supported_version()
-        return self.version_lock.get_locked_version(self.id, min_stack)
+        """Retrieve the latest known version of the rule, considering the version lock setting."""
+        toml_version = self.data.get("version")
+
+        if BYPASS_VERSION_LOCK and toml_version:
+            return toml_version
```

Comment:
This is where the question of a default applies. 

For option 3: _defer to kibana_ this _should_ be the only time this is `None`. However, I imagine the api treats `version: null` different from not setting at all.

So the question is, should we raise an error here, and then convey that `include_version` should NOT be set if you fit option 3

Reply:
enforce on import or export (controlled by Export variable) `kibana export-rules --strip-version-revision`

Reply:
Just realized that we already did this. If a user wants to use kibana versions, they won't strip.

Reply:
I also had to set version (defaulted to `1` since `null` is not accepted on import)